search/__init__.py

```diff
@@ -8,4 +11,11 @@ def create_app() -> Flask:
     app = Flask(__name__)
     app.register_blueprint(get_search_blueprint())
     app.register_blueprint(get_ping_blueprint())
+
+    if app.config.get('ENV') == 'development':
```

Comment:
this is hardcoding the name of one environment, so it may make difficult to reuse the application across custom environments (but it's just `development`); I think we are after a setting named something like `APP_ERROR_BLUEPRINT` or `APP_ERROR_ROUTES` that can be turned on or off. This can be turned on in `development` and be off by default.

How do we turn on a setting in development? By putting a `env:` line in `docker-compose-dev.yml` that passes in a setting.

Reply:
There is a config setting entitled: 'TESTING' which is automatically set to True when the tests are run. I'll update the code to use this as the `/error` endpoint was only implemented as method to test error responses.

Reply:
I have moved the registration of the error blueprint to the pytest fixture where the test client is created, removing the need to check a config variable.